cloud/aws-common/pom.xml

```diff
@@ -46,6 +46,10 @@
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-java-sdk-s3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-sts</artifactId>
+        </dependency>
```

Comment:
let's add a comment to explain why this dependency is needed, since it might otherwise accidentally get removed if there is no compile-time dependency on it.

We also need to add the same dependency to the `druid-aws-rds-extensions` module, since that module does not depend on aws-common and instead pulls in its own set of aws libraries.

Reply:
AFAIK `druid-aws-rds-extensions` does not need `aws-java-sdk-sts` as a dependency since it does not use STS?

Reply:
Yes but the error message is from the same [AWSCredentialUtils](https://github.com/confluentinc/druid/blob/0.22.1-confluent/cloud/aws-common/src/main/java/org/apache/druid/common/aws/AWSCredentialsUtils.java) class, so adding it under aws-common should be enough?